source/exe/process_wide.cc

```diff
@@ -15,10 +16,20 @@ namespace {
 std::atomic<uint32_t>& processWideInitialized() {
   MUTABLE_CONSTRUCT_ON_FIRST_USE(std::atomic<uint32_t>);
 };
+
+// Per absl documentation it is safe for this to be a namespace-scoped global variable.
+absl::once_flag create_once;
```

Comment:
can this be static or in an anon namespace?

Reply:
It is in an anon namespace I think?